pkg/kubectl/resource/builder.go

```diff
@@ -198,7 +198,7 @@ func (b *Builder) Unstructured() *Builder {
 		b.errs = append(b.errs, fmt.Errorf("no unstructured mapper provided"))
 		return b
 	}
-	if b.mapper != nil {
+	if b.mapper != nil && b.mapper != b.unstructured {
```

Comment:
Does it mean that there is no error even if internal mapper is set?

Reply:
no, it means we tolerate a second call to `Unstructured()`

Reply:
The fix works only because two calls do NOT share the same Builder as previous (changes in `describe.go`).

I think the change in `builder.go` is unnecessary and it changes current behavior.

You can assume that the first call set `b.mapper = b.unstructured`, so if using the same builder the second call will meet the condition and get the error.